### DISCUSSION TITLE
Bądźmy eko

### DISCUSSION THREAD

**Discussion by hawkeye116477** [MajkiIT/polish-ads-filter#17216] [Ideas] [answered]
Jest sprawa do dyrka @MajkiIT. Obecnie GitHub Actions uruchamia się po każdym push/merge, idealnie byłoby dobrze jakby działało to jedynie po dokonaniu zmian do strony. Więc otóż w ramach akcji eko trzeba utworzyć nową gałąź `gh-pages` albo `website`, a potem wejść w opcje repo i przestawić „GitHub Pages Source" na nową gałąź (katalog niezmiennie docs). A jako ostatni krok w https://github.com/MajkiIT/polish-ads-filter/blob/master/.github/workflows/hugo.yml zmienić wszystkie wystąpienia `master` na nazwę nowej gałęzi `gh-pages`.

**Comment by xxcriticxx** [comment]
by sie przydal numerek przy tych rozmowach tak samo jak jest przy issues

**Comment by hawkeye116477** [comment]
@xxcriticxx Przecież jest numer #17216.

**Comment by xxcriticxx** [comment]
![Capture](https://user-images.githubusercontent.com/15007183/94365468-54f3dc80-009f-11eb-9b38-acbf3d2f6d05.PNG)
chodzi mi o ten numer wtedy wiadomo ze sa jakies nowe rozmowy

**Comment by hawkeye116477** [comment]
@xxcriticxx No to możesz napisać im co do poprawy, itd na https://support.github.com/contact/feedback?category=discussions :smile:

**Comment by hawkeye116477** [comment, marked as answer]
Fixed #17259, #17260, #17262.

**Comment by hawkeye116477** [comment]
@xxcriticxx Napisałem do nich o tym, to oni na to, że był licznik, ale usunęli, bo @xxcriticxx powiedział, że mu przeszkadza...

> Thanks for this feedback. We removed the count based on feedback from @xxcriticxx but have been looking into other ways to indicate recent activity within this tab/area of the repository.
